web/pages/meetings/office_hours.md

```diff
@@ -0,0 +1,20 @@
+title: PlasmaPy "Office" Hours
+author: Nick Murphy
+hidetitle: True
+
+[Zoom link]: https://harvard.zoom.us/j/98483606703?pwd=cDA0em5VMEk1cUd0RXhmRUJBVzNiQT09
+[Matrix chat room]: https://app.element.io/#/room/#plasmapy:openastronomy.org
+
+# PlasmaPy "Office" Hours
+
+Do you have a question about PlasmaPy?
+Would you like to learn how to become active in the PlasmaPy project?
+Do you have an idea for a great new feature or want to start an affiliated package?
+If so, please join us for **informal PlasmaPy "office" hours** at the following times:
+
+ - Wednesday, October 7 from 2–3 pm EDT
+ - Thursday, October 8 from 2–3 pm EDT
+
+We'll meet using this [Zoom link] (passcode 103519).
```

Comment:
Are you sure you want to publish a passcode?  This just seems to be asking for trouble.  It might be better to have a waiting room for zoom or have them come to the chat to get a passcode.  It's your zoom, so it's your call.

Reply:
Yeah, I was wondering about that.  I had forgotten about the waiting room feature though, so that sounds like a good alternative.